astrodendro/io/hdf5.py

```diff
@@ -41,21 +41,31 @@ def dendro_export_hdf5(d, filename):
     ds.attrs['IMAGE_VERSION'] = '1.2'
     ds.attrs['IMAGE_MINMAXRANGE'] = [d.data.min(), d.data.max()]
 
+    try:
+        f.create_dataset('wcs_header', data=d.wcs.to_header_string())
+    except AttributeError:
+        pass
+
     f.close()
 
 
 def dendro_import_hdf5(filename):
     """Import 'filename' and construct a dendrogram from it"""
     import h5py
+    import astropy.wcs
```

Comment:
I have a slight preference for `from astropy.wcs import WCS` (a few less "wcs" to type)